packages/module/src/ChatbotModal/ChatbotModal.scss

```diff
@@ -0,0 +1,93 @@
+.pf-chatbot__chatbot-modal-backdrop {
+  position: static;
+}
+
+.pf-chatbot__chatbot-modal {
+  --pf-v6-c-modal-box--BorderRadius: var(--pf-t--global--border--radius--medium);
+  position: fixed;
+  inset-block-end: var(--pf-t--global--spacer--800); // no associated semantic token
+  inset-inline-end: var(--pf-t--global--spacer--lg);
+  width: 30rem;
+  height: 70vh;
+  background-color: var(--pf-t--global--background--color--secondary--default);
+
+  .pf-v6-c-modal-box__title {
+    --pf-v6-c-modal-box__title--FontSize: var(--pf-t--global--font--size--heading--h3);
+  }
+  .pf-v6-c-button.pf-m-primary.pf-m-block,
+  .pf-v6-c-button.pf-m-link.pf-m-block {
+    --pf-v6-c-button--FontWeight: 500;
```

Comment:
Is there a semantic token for this?

Reply:
Unfortunately not for 500, as far as I can tell. 
<img width="689" alt="Screenshot 2024-11-18 at 12 02 08 PM" src="https://github.com/user-attachments/assets/63035ac1-bdd9-45bf-b60b-635d451f27aa">